ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/requirements.txt

```diff
@@ -1,4 +1,4 @@
 -i https://pypi.org/simple
 /opt/dataflow/prepackaged_df-1.0-py3-none-any.whl
-oci==2.74.0
+/opt/dataflow/oci-2.70.1+preview.1.1420-py2.py3-none-any.whl
```

Comment:
Do we really need to specify this version at all?

Reply:
I'm not sure when Async will be GA. Right now we need to package the wheel since the preview SDK isn't available in public repositories